audbackend/core/filesystem.py

```diff
@@ -75,32 +89,46 @@ def _get_file(
 
     def _ls(
             self,
-            folder: str,
+            path: str,
     ):
-        r"""List all files under folder."""
+        r"""List all files under (sub-)path."""
+
+        if path.endswith('/'):  # find files under sub-path
+
+            path = self._expand(path)
+            if not os.path.exists(path):
+                utils.raise_file_not_found_error(path)
+            paths = audeer.list_file_names(path, recursive=True)
```

Comment:
I think we should also allow for hidden file names (and should add a test for them):

```suggestion
            paths = audeer.list_file_names(path, recursive=True, hidden=True)
```

Reply:
I added tests for hidden files.